packages/frontend/app/components/reports/subject/course.hbs

```diff
@@ -1,20 +1,23 @@
-<Reports::SubjectHeader
-  @report={{@report}}
-  @school={{@school}}
-  @subject={{@subject}}
-  @prepositionalObject={{@prepositionalObject}}
-  @prepositionalObjectTableRowId={{@prepositionalObjectTableRowId}}
-  @year={{@year}}
-  @changeYear={{@changeYear}}
-  @showYearFilter={{not-eq @prepositionalObject "academic year"}}
-  @description={{@description}}
-  @fetchDownloadData={{this.fetchDownloadData}}
-  @readyToDownload={{this.data.isResolved}}
-/>
+{{#if this.allCoursesData.isResolved}}
```

Comment:
I'm not seeing the reason to hide this here, especially with the `@readyToDownload` argument that prevents any shenanigans. I, for sure, don't think we need both.